recipes/redbiom/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "redbiom" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b5a1f8a8e6d81159fbe639cffd2aff0abffc769247997ff19f1f36755c1f9da7
+
+build:
+  skip: true  
```

Through this, you aren't building anything at the moment, package can be noarch, so better replace this with:
```suggestion
  noarch: python
```